terraform/groups/lambda/main.tf

```diff
@@ -0,0 +1,70 @@
+provider "aws" {
+  region = var.aws_region
+}
+
+terraform {
+  backend "s3" {
+    encrypt = true
+  }
+}
+
+provider "vault" {
+  auth_login {
+    path = "auth/userpass/login/${var.vault_username}"
+    parameters = {
+      password = var.vault_password
+    }
+  }
+}
+
+data "terraform_remote_state" "network_remote_state" {
+  backend = "s3"
+  config = {
+    bucket = var.remote_state_bucket
+    key    = var.remote_state_key
+    region = var.aws_region
+  }
+}
+
+locals {
+  test_and_development_vpc_id     = data.terraform_remote_state.network_remote_state.outputs.vpc_id
```

Comment:
We might have an easier time here retrieving the vpc id using profile variables. (unsure whether we want to build into later environments?)